app/client/src/components/pages/Community.Tabs.Monitoring.js

```diff
@@ -56,11 +50,51 @@ const containerStyles = css`
   }
 `;
 
+const modifiedDisclaimerStyles = css`
+  ${disclaimerStyles};
+  padding-bottom: 0;
+`;
+
 const modifiedErrorBoxStyles = css`
   ${errorBoxStyles};
   text-align: center;
 `;
 
+const modifiedToggleTableStyles = css`
+  ${toggleTableStyles};
+  tfoot {
+    background-color: #f0f6f9;
+
+    tr:first-of-type {
+      border-top: 2px solid #dee2e6;
+      th {
+        border: none;
+        &:first-of-type {
+          display: inline-block;
+          margin-left: 46px;
+        }
+      }
+    }
+    .download-links {
+      border-bottom: 1px solid #dee2e6;
+      th {
+        border-top: none;
+        div {
+          display: inline-flex;
+          gap: 3px;
+          width: 50%;
+          &:first-of-type {
+            justify-content: flex-start;
+          }
+          &:last-of-type {
+            justify-content: flex-end;
+          }
+        }
```

Comment:
These styles are causing some space between the "Advanced Filtering" text and icon on larger screens. Can you look into this? 

![image](https://user-images.githubusercontent.com/46329268/168288944-349cce90-783b-4c25-9278-0aeb29cb6258.png)

Reply:
The spacing issue looks fixed now. Thank you!